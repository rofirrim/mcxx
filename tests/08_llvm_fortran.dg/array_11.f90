! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="array_11.f90.out"
! </testinfo>
PROGRAM MAIN
    IMPLICIT NONE
    INTEGER :: A(3), B(3)
    LOGICAL :: L(3)

    A(1) = 1
    A(2) = 2
    A(3) = 3

    B(1) = 3
    B(2) = 2
    B(3) = 1

    L = A == B
    PRINT *, L

    L = A /= B
    PRINT *, L

    L = A > B
    PRINT *, L

    L = A < B
    PRINT *, L

    L = A >= B
    PRINT *, L

    L = A <= B
    PRINT *, L
END PROGRAM MAIN
