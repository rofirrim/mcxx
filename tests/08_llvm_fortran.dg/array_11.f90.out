 F T F
 T F T
 F F T
 T F F
 F T T
 T T F
